ecs-cli/modules/cli/local/localproject/project.go

```diff
@@ -217,6 +218,11 @@ var readTaskDefFromRemote = func(remote string, p *LocalProject) (*ecs.TaskDefin
 
 	commandConfig, err := newCommandConfig(p.context, rdwr, region)
 	if err != nil {
+		logrus.WithFields(logrus.Fields{
+			"error": err,
+			"region": region, // Useful for debugging region resolution across partitions, e.g.
```

Comment:
I think the region field is only passed in when it's read through the ARN. It'd be nice to get whatever region it tried to use in the CommandConfig. 

(I tried producing the error locally and the output was: 
`FATA[0000] Unable to create local project                error="UnrecognizedClientException: The security token included in the request is invalid.\n\tstatus code: 400, request id: 41104651-a3a5-4189-a0ef-411d6a82
950d"`).

Reply:
Since this is something that will only surface if using creds from one partition for another (which I'm assuming is fairly rare), I think we can maybe follow up on this later as an enhancement.